runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -517,7 +517,8 @@ def CreateGdbConsole(dict: dict<any>, pty: string, commpty: string): string
   endwhile
 
   if !success
-    return 'Failed to startup the gdb program.'
+    return $'Failed to startup the gdb program within {counter_max}ms, consider increasing the timeout'
+      .. counter_max .. ' x 10ms, may consider to increase timeout'
```

Comment:
This seems to have some issues.
1. There is an extra/leftover line
2. The message is inaccurate (it assumes counter_max is in milliseconds)
3. Missing '.'

```suggestion
    return $'Failed to startup the gdb program within {counter_max} x 10ms, consider increasing the timeout.'
```

Reply:
Yep, I accepted the change without checking carefully.
Now, I saw that someone already added timeout info in the terminal.txt for termdebug. And @LemonBoy mentioned that counter by 10ms is a bit strange, now I'm thinking might as well change the counter to ms?

I will withdraw this PR, and file another one.

I'm not very familiar with github PR flow, sorry for the inconvenience.

Reply:
I guess the granularity is 10ms to avoid having a busy polling loop here. The docs should be slightly amended to mention `timeout` is a timeout in terms of `10ms` ticks (or units).

Reply:
Well, counter can be made to +10 instead of +1 to align with ms. I think this way is less confusing.

Reply:
> mentioned that counter by 10ms is a bit strange, now I'm thinking might as well change the counter to ms?

Is that an incompatible change? Something that breaks existing config files? I wouldn't couple that with a help change.

No reason to open a different PR. Changes are made as needed and usually commits are squashed and/or force updated as things proceed.

Reply:
Something like `min([1, g:termdebug_config['timeout'] / 10])` would also work, I doubt anyone would notice the timeout is rounded to 10ms.

Reply:
> > mentioned that counter by 10ms is a bit strange, now I'm thinking might as well change the counter to ms?
> 
> Is that an incompatible change? Something that breaks existing config files? I wouldn't couple that with a help change.
> 
> No reason to open a different PR. Changes are made as needed and usually commits are squashed and/or force updated as things proceed.

Yes, it will be existing configuration timeout /10 as intended, if timeout early, there is a problem, the "debugged program" terminal window will not show, menu will not show etc.

Hmm, if going this way, this PR is not really necessary any more, someone already added the timeout section in the doc, and the error message anyway is not visible without -V option and check the log afterwards.